Gemfile

```diff
@@ -49,6 +46,8 @@ group :development do
   gem 'spring'
   gem 'spring-commands-rspec'
   gem 'spring-watcher-listen', '~> 2.0.0'
+
+  gem 'foreman'
```

Comment:
No, please do not pull in foreman again. I removed it back then because having multiple processes running in a single docker service caused troubles when debugging the app.

Reply:
Which issues exactly? Having to (re-)start the watchers for JS and CSS is super annoying. Using foreman is the default setup with Rails.

Reply:
The issue of `binding.pry` interactively through `docker-compose`. The keystrokes would hit foreman but how should it know which process is the receiver?